api/tests/integration/infrastructure/repositories/schooling-registration-repository_test.js

```diff
@@ -142,6 +142,27 @@ describe('Integration | Infrastructure | Repository | schooling-registration-rep
     });
   });
 
+  describe('#findByUserIdAndSCOOrganization', () => {
+
+    it('should return schoolingRegistrations belonging to SCO organizations', async () => {
```

Comment:
```suggestion
    it('should return schoolingRegistrations belonging to SCO organizations and user only', async () => {
```

Si on s'en tient strictement au nom du test, la méthode pourrait récupérer les schooling registrations de tous les élèves de l'orga.
Ça vaut peut-être le coup de faire un deuxième test ou rajouter des données pour s'assurer qu'on ne récupère que celles du user indiqué

Reply:
J'ai rajouté un autre user et une schooling-registration associée dans le même test.